odoo/addons/test_lint/tests/test_docstring.py

```diff
@@ -38,13 +42,14 @@
 }
 
 
-def extract_docstring_types(method):
-    docstring = inspect.cleandoc(method.__doc__)
-    doctree = docutils.core.publish_doctree(docstring, settings_overrides={
-        'report_level': 5,
-        'halt_level': 5,
-    })
+def role_function(name, rawtext, text, lineno, inliner, options=None, content=None):
+    return [docutils.nodes.inline(rawtext, text)], []
 
+for role in ('attr', 'class', 'func', 'meth', 'ref', 'const', 'samp', 'term'):
```

Comment:
Is this normal that you register parsers in the body of the module? Why not inside the test class setup?